modules/images/webp-uploads/image-edit.php

```diff
@@ -120,7 +120,7 @@ function ( $metadata, $post_meta_id ) use ( $post_id, $file_path, $mime_type, $e
 
 			$old_metadata = wp_get_attachment_metadata( $post_id );
 			$resize_sizes = array();
-			$target       = isset( $_REQUEST['target'] ) ? sanitize_key( $_REQUEST['target'] ) : 'all';
+			$target       = isset( $_REQUEST['target'] ) ? sanitize_key( $_REQUEST['target'] ) : 'all'; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
It may be worth adding a comment to explain why this error is being silenced here, especially because this is security related:
```suggestion
			// A nonce check is not necessary here as this logic directly ties in with WordPress core logic which already has one.
			$target       = isset( $_REQUEST['target'] ) ? sanitize_key( $_REQUEST['target'] ) : 'all'; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```